### PR TITLE
Update notes patch spec to use json-patch

### PR DESCRIPTION
**Pull request by in-op**

### CODE REVIEW

File: src/endpoints/project_notes.yaml [hunk 1]
```diff
@@ -106,7 +106,7 @@ paths:
       requestBody:
         description: JSON Patch document as defined by [RFC 6902](https://www.rfc-editor.org/rfc/rfc6902)
```
Comment (dave-shawley) [inline src/endpoints/project_notes.yaml:107]: d'oh ... change `requestBody` to reference [components/requests.yaml#/jsonPatch](https://github.com/AWeber-Imbi/imbi-openapi/blob/3292150db38f81284a107de4c871da8e77776dc0/src/components/requests.yaml#L1-L15) instead.
```yaml
      requestBody:
        $ref: '../components/requests.yaml#/jsonPatch'
```

Comment (in-op) [reply src/endpoints/project_notes.yaml]: Ah. Updated

Comment (dave-shawley) [reply src/endpoints/project_notes.yaml]: bah ... i typoed :/

Comment (dave-shawley) [reply src/endpoints/project_notes.yaml]: nvm ... you typoed

File: src/endpoints/project_notes.yaml [hunk 2]
```diff
@@ -104,15 +104,7 @@ paths:
...
-        description: JSON Patch document as defined by [RFC 6902](https://www.rfc-editor.org/rfc/rfc6902)
-        content:
-          application/json-patch+json:
-            schema:
-              $ref: '../components/schemas.yaml#/jsonPatch'
-            example:
-              - op: replace
-                path: "/content"
-                value: "Bacon ipsum dolor amet alcatra tongue shankle, beef ribs ribeye filet mignon drumstick landjaeger capicola andouille pancetta meatloaf cupim bresaola rump. Pork belly ham turkey, salami prosciutto ribeye beef ribs swine ground round pork loin leberkas tri-tip pig cow pastrami. Prosciutto tri-tip strip steak cow. Shankle hamburger kevin, pig meatloaf burgdoggen kielbasa ball tip pastrami tongue sirloin corned beef turducken flank porchetta. Filet mignon bacon chislic, chicken burgdoggen cow pig. Landjaeger chicken jowl kielbasa bresaola bacon. Spare ribs turkey prosciutto boudin pastrami pancetta leberkas."
+        $ref: '../components/requests.yaml#jsonPatch'
```
Comment (dave-shawley) [inline src/endpoints/project_notes.yaml:107]: Missing the leading slash (root) in the fragment
```
$ref: '../components/requests.yaml#/jsonPatch'
```

**Pull request merged**